store/postgres/src/relational_queries.rs

```diff
@@ -855,17 +861,18 @@ impl<'a> QueryFilter<'a> {
         attribute: &Attribute,
         value: &Value,
         negated: bool,
+        strict: bool,
         mut out: AstPass<Pg>,
     ) -> QueryResult<()> {
         let column = self.column(attribute);
-
+        let operation = if strict { "like" } else { "ilike" };
```

Comment:
Better would be to add this only to the `Value::String(s)` match arm:
```
let operation = match (strict, negated) {
  (true, true) => " not like ",
  (true, false) => " like ",
 (false, true) => " not ilike ",
 (false, false) => " ilike ",
};
```
and then have `out.push_sql(operation)` instead of the `if negated { .. }` statement. That avoids allocations - this code is run several times for each query, and tries to be as careful with allocations as possible.